app/src/lib/stats/stats-database.ts

```diff
@@ -607,6 +607,21 @@ export interface IDailyMeasures {
   /** The number of times the user applies the "Included in Commit" Filter */
   appliesIncludedInCommitFilterCount: 0
 
+  /** The number of times the user applies the "Excluded from Commit" Filter */
+  appliesExcludedFromCommitFilterCount: 0
+
+  /** The number of times the user applies the "New Files" Changes Filter */
+  appliesNewFilesChangesFilterCount: 0
+
+  /** The number of times the user applies the "Modified Files" Changes Filter */
+  appliesModifiedFilesChangesFilterCount: 0
+
+  /** The number of times the user applies the "Deleted Files" Changes Filter */
+  appliesDeletedFilesChangesFilterCount: 0
+
+  /** The number of times the user clears all filters */
+  appliesClearAllFiltersCount: 0
```

Comment:
```suggestion
  appliesClearAllChangesListFilterCount: 0
```

^ Apply this as needed to make this stat unique.